src/views/pages/hardware/management.ejs

```diff
@@ -0,0 +1,166 @@
+                            </form>
+                          </div>
+                        </div>
+                      </div>
+                    </div>
+              </div>
+            </div>
+            <div class="col-md-8">
+              <div class="card">
+                <div class="card-header card-header-tabs card-header-success">
+                  <h4 class="card-title">Hardware overview</h4>
+                </div>
+                <div class="card-body">
+                  <table class="table">
+                    <thead class="text-success">
+                      <th></th>
+                      <th>Item</th>
+                      <th>Total (Available) Stock</th>
+                      <th>Taken (Reserved) Stock</th>
+                      <th>Taken By</th>
+                      <th>Reserved By</th>
+                    </thead>
+                    <tbody>
+                      <% items.forEach(item => { %>
+                        <tr>
+                          <td>
+                            <button class="btn-just-icon" onclick='deleteItemConfirm(<%= item.id %>)'>
+                              <i class="fas fa-trash-alt delete-btn"></i>
+                            </button>
+                            <a href='javascript:editItem(<%= JSON.stringify(item) %>)'>
+                              <i class="fas fa-pen edit-btn"></i>
+                            </a>
+                          </td>
+                          <td>
+                            <%= item.name %>
+                          </td>
+                          <td>
+                            <%- item.totalStock %> (<%- item.totalStock - item.reservedStock - item.takenStock %>)
+                          </td>
+                          <td>
+                            <%- item.takenStock %> (<%- item.reservedStock %>)
+                          </td>
+                          <td>
+                            <% item.reservations.filter(reservation => reservation.isReserved === false).forEach(reservation => { %>
+                              <a href="/user/<%- reservation.user.id %>"><%- reservation.user.name %></a>,
+                            <% }) %>
+                          </td>
+                          <td>
+                            <% item.reservations.filter(reservation => reservation.isReserved === true).forEach(reservation => { %>
+                              <a href="/user/<%- reservation.user.id %>"><%- reservation.user.name %></a>,
+                            <% }) %>
+                          </td>
+                        </tr>
+                      <% }) %>
+                    </tbody>
+                  </table>
+                </div>
+              </div>
+            </div>
+          </div>
+        </div>
+        <script src="/js/hardwareManagement.js"></script>
+        <script>
+          <% items.forEach(item => { %>
+            items["<%= item.id %>"] = {
```

Comment:
I think this code can be removed, it might be left over from debugging

Reply:
I've cleaned it up a little bit but I can't remove it since it's used for the confirmation modal when you try to delete any item.
Because the confirmation modal gets rendered by front-end JS we need to have the items in a front-end JS variable. I add the items with a loop because I want them to be stored in a map where the key is the id of the item.

Reply:
Looks good now